core/modules/field/field.module

```diff
@@ -1308,6 +1309,72 @@ function field_get_block_list() {
   return $types;
 }
 
+/**
+ * Helper function to retrieve the most commonly used label for a field.
+ *
+ * @param string $machine_name
+ *   The machine name for the field in question.
+ *
+ * @return string
```

Comment:
If FALSE can be returned it should say `@return string|FALSE` or something like that. Or just have it as `return;`